crates/codegen/parser/runtime/src/kinds.rs

```diff
@@ -13,11 +13,16 @@ use napi_derive::napi;
     strum_macros::Display,
     strum_macros::EnumString,
 )]
+#[strum(use_phf)]
 #[cfg_attr( feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "kinds") )]
 #[cfg_attr(not(feature = "slang_napi_interfaces"), derive(Clone, Copy))]
 pub enum TokenKind {
     SKIPPED,
-    // Expanded by the template engine
+    // Used for testing ithis crate, this is generated in the client code
+    Identifier,
+    X,
```

Comment:
nit: what do you think about using `Token1`, `Token2`, `Token3` instead of `X`, `Y`, and `Z`?
And `Rule1`, `Rule2`, `Rule3` instead of `A`, `B`, `C`?

I think it would make it a bit clearer/easier to me to understand the query tests, if the variant we explicit here. It is worth a few extra characters.

Reply:
Done